### ISSUE TITLE
cf org <org name> doesn't work in 6.18.1

### ISSUE THREAD

**Issue by saliceti** [cloudfoundry/cli#854]
### Command
`cf org <org name>` should return the organisation details but they are empty in 6.18.1

### CLI Version
6.18.1

### Error

With `cf` 6.18.0 and before:
```
$ cf org paas_user_research
Getting info for org paas_user_research as admin...
OK

paas_user_research:                   
                      domains:        cloudapps.digital
                      quota:          paas_demo (10240M memory limit, Unlimited instance memory limit, 1000 routes, 10 services, paid services allowed, unlimited app instance limit)
                      spaces:         sandbox
                      space quotas:   
```
With `cf` 6.18.1:
```
14:35:31:aws:ruby-2.2.3:bin(master)$ cf org paas_user_research
Getting info for org paas_user_research as admin...
OK

paas_user_research:                   
                      domains:        
                      quota:           (0M memory limit, 0M instance memory limit, 0 routes, 0 services, paid services disallowed, unlimited app instance limit)
                      spaces:         
                      space quotas:   
```

### CC API Endpoint Version
`API version: 2.54.0`

### CF Trace
```
$ CF_TRACE=true cf org paas_user_research

REQUEST: [2016-06-08T14:40:13+01:00]
GET /v2/organizations?q=name%3Apaas_user_research HTTP/1.1
Host: api.cloud.service.gov.uk
Accept: application/json
Authorization: [PRIVATE DATA HIDDEN]
Content-Type: application/json
User-Agent: go-cli 6.18.1+a1103f0 / darwin



RESPONSE: [2016-06-08T14:40:13+01:00]
HTTP/1.1 200 OK
Content-Length: 1530
Connection: keep-alive
Content-Type: application/json;charset=utf-8
Date: Wed, 08 Jun 2016 13:40:13 GMT
Server: nginx
X-Content-Type-Options: nosniff
X-Vcap-Request-Id: 9eed4c83-4d83-4981-bb71-6aa7bf08c8c3

{
  "total_results": 1,
  "total_pages": 1,
  "prev_url": null,
  "next_url": null,
  "resources": [
    {
      "metadata": {
        "guid": "49a23332-8cec-43d8-837c-89098d570c70",
        "url": "/v2/organizations/49a23332-8cec-43d8-837c-89098d570c70",
        "created_at": "2016-06-08T12:54:10Z",
        "updated_at": "2016-06-08T13:10:58Z"
      },
      "entity": {
        "name": "paas_user_research",
        "billing_enabled": false,
        "quota_definition_guid": "f9bbc5cb-1d8f-4532-bb0f-5b556e581ae8",
        "status": "active",
        "quota_definition_url": "/v2/quota_definitions/f9bbc5cb-1d8f-4532-bb0f-5b556e581ae8",
        "spaces_url": "/v2/organizations/49a23332-8cec-43d8-837c-89098d570c70/spaces",
        "domains_url": "/v2/organizations/49a23332-8cec-43d8-837c-89098d570c70/domains",
        "private_domains_url": "/v2/organizations/49a23332-8cec-43d8-837c-89098d570c70/private_domains",
        "users_url": "/v2/organizations/49a23332-8cec-43d8-837c-89098d570c70/users",
        "managers_url": "/v2/organizations/49a23332-8cec-43d8-837c-89098d570c70/managers",
        "billing_managers_url": "/v2/organizations/49a23332-8cec-43d8-837c-89098d570c70/billing_managers",
        "auditors_url": "/v2/organizations/49a23332-8cec-43d8-837c-89098d570c70/auditors",
        "app_events_url": "/v2/organizations/49a23332-8cec-43d8-837c-89098d570c70/app_events",
        "space_quota_definitions_url": "/v2/organizations/49a23332-8cec-43d8-837c-89098d570c70/space_quota_definitions"
      }
    }
  ]
}
Getting info for org paas_user_research as admin...
OK

paas_user_research:                   
                      domains:        
                      quota:           (0M memory limit, 0M instance memory limit, 0 routes, 0 services, paid services disallowed, unlimited app instance limit)
                      spaces:         
                      space quotas:   
```
### Platform Details
Mac OS X 10.11.5 and Ubuntu 14.04.4 64-bit

### Shell
Terminal

**Comment by dkoper** [comment]
Hi @saliceti

We fixed this in the following story, will be included in the next release:
https://www.pivotaltracker.com/story/show/119658109

Regards,
Dies Koper
CF CLI PM

**Comment by dkoper** [comment]
We have fixed this in cf CLI 6.19.0, released today.
https://github.com/cloudfoundry/cli#downloads

Regards,
Dies Koper
CF CLI PM

**Comment by saliceti** [comment]
I confirm this is working now. Thanks!
